htdocs/core/class/html.formticket.class.php

```diff
@@ -412,6 +412,7 @@ function(response) {
         $reshook = $hookmanager->executeHooks('formObjectOptions', $parameters, $ticketstat, $action); // Note that $action and $object may have been modified by hook
         if (empty($reshook))
         {
+        	$ticketstat->array_options = array_merge($ticketstat->array_options,$this->array_options);
```

Comment:
ce n'est pas normal d'avoir un $this->array_options dans une class form, il faudrait voir si via le hook tu peux pas peupler ticketstat plutot